Elements/src/Geometry/Line.cs

```diff
@@ -850,6 +880,49 @@ public Line ExtendTo(Polygon polygon, double maxDistance, bool bothSides = true,
             return ExtendTo(polygon.Segments(), maxDistance, bothSides, extendToFurthest, tolerance);
         }
 
+        /// <summary>
+        /// Measures the distance between two lines.
+        /// Computes a cross product of the lines' directional vectors.
+        /// Case 1: It's zero, meaning the lines are collinear, which means we can simply orthogonalize a vector that connects points on different line wrt the directional vector.
```

Comment:
The comments are great but I realized that there are not much value for API caller from them. Lets distribute the through the implementation above key code lines. Some comments will became obsolete after that like "Computes a cross product of the lines' directional vectors." because it's obvious. Lines as #900 and #917 would need explanation as it's 100% impossible to understand what's going on othervise.